src/svyby.jl

```diff
@@ -37,3 +37,14 @@ function svyby(formula::Symbol, by::Symbol, design::AbstractSurveyDesign, func::
     gdf = groupby(design.data, by)
     return combine(gdf, [formula, :weights] => ((a, b) -> func(a, design, b, params...)) => AsTable)
 end
+
+"""
+    svyby(formula, by, design, function, params)
+
+Generate subsets of a StratifiedSample.
+"""
+function svyby(formula::Symbol, by::Symbol, design::StratifiedSample, func::Function)
+    # TODO: add functionality for `formula::AbstractVector`
+    gdf_domain = groupby(design.data, by)
+    return combine(gdf_domain, [formula, :sampfraction, design.strata] => ((a,b,c) -> func(a,b,c,design)) => AsTable ) 
```

Comment:
`combine` here doesn't take `weights` as argument anymore, but we might need it for `svytotal`. We'll see when we implement `svyby` for `svytotal`.

Reply:
Yes that is possible